hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/ServerCrashProcedure.java

```diff
@@ -474,6 +474,7 @@ protected boolean isMatchingRegionLocation(RegionStateNode rsn) {
    * it.
    */
   private void assignRegions(MasterProcedureEnv env, List<RegionInfo> regions) throws IOException {
+    LOG.info("=============assign " + regions);
```

Comment:
Need this? Don't we have enough spew in log already around assign?

Reply:
Oh, this is a debug log. Let me remove.